layouts/partials/collapsibleTags.html

```diff
@@ -0,0 +1,32 @@
+<details>
+  {{ $s := slice }}
+  {{ range .WeightedPages }}
+    {{ $s = $s | append .Title }}
+  {{ end }}
+  {{ $set := uniq $s }}
+  {{ $len := len $set }}
+  <summary>
+    <h3 style="display: inline; color: #2E8B57;">{{ .Page.Title }}&nbsp</h3>
+    <h3 style="display: inline;">{{- T "group" $len -}}</h3>
+  </summary>
+    <ul>
+      {{ range .Pages.GroupBy "Title" }}
+      {{ $len := len .Pages }}
+      <details>
+        <summary>
+          <h3 style="display: inline; color: #2E8B57;">{{ .Key}}&nbsp</h3>
+          <h3 style="display: inline;">{{- T "article" $len -}}</h3>
+        </summary>
+        <br>
+        {{range .Pages}}
+          <ul>
+            <li>
+              <a href="{{ .RelPermalink }}">{{ substr .Summary 0 20}}...</a>
```

Comment:
Should the `.Summary` needs to be added? I think just the post title is enough.

Reply:
I agree.
In Idolstation's case, the .Title of a md file is actually the name of the Twitter account while .Summary is like the actual title of the tweet.
e.g. if .Title
<img width="377" alt="Screen Shot 2021-06-18 at 7 43 13 PM" src="https://user-images.githubusercontent.com/20605335/122556092-ae58cd00-d06d-11eb-842d-00d407c56ec0.png">
if .Summary
<img width="464" alt="Screen Shot 2021-06-18 at 7 42 37 PM" src="https://user-images.githubusercontent.com/20605335/122556105-b31d8100-d06d-11eb-8135-2c12516844f7.png">

Reply:
Ok, but the theme is not just for the single special site. The way you aggregate Twitter title is a targeted solution (summary is not equal to subtitles). I suggest making this a configurable param? Let users choose whether to render the summary.

Also, it's needed to parse the substring with `safeHTML`:

```html
{{ safeHTML (substr .Summary 0 20) }}
```